.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us fix your issue
+
+---
+
+**Is this the right place for my bug report?**
+This repository contains the Linux kernel used on the Raspberry Pi. If you believe that the issue you are seeing is kernel related, this is the right place. If not, we have other repositories for the GPU firmware (https://github.com/raspberrypi/firmware) and Raspberry Pi userland applications (https://github.com/raspberrypi/userland). If you simply have a question, then the Raspberry Pi forums are the best place for that (https://www.raspberrypi.org/forums).
+
```

Comment:
Maybe also add that problems with distribution packages should be reported here: https://github.com/RPi-Distro/repo

Reply:
Done